### PR TITLE
Define initial traffic-generator.yang model

### PR DESCRIPTION
**Pull request by purplezimmermann**
And add dependency `ietf-yang-types@2013-07-15.yang`

### CODE REVIEW

**Review by vlvassilev — DISMISSED** [review]
Maybe skip standard models.

File: yang/traffic-generator.yang [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        "Number of octets to wait between ethernet frames.";
+    }
+
+    leaf source-address {
+      type yang:mac-address;
+      description
+        "Source MAC address in ethernet frame headers.";
+    }
+
+    leaf destination-address {
```
Comment (vlvassilev) [inline yang/traffic-generator.yang]: MAybe shorten destination to dst.

**Comment by purplezimmermann** [comment]
@vlvassilev: Deleted `ietf-yang-types@2013-07-15.yang` from the commit

File: yang/traffic-generator.yang (hunk 1 above)
Comment (purplezimmermann) [reply yang/traffic-generator.yang]: @vlvassilev: Addressed! And also shortened `source-address` to `src-address`

**Comment by purplezimmermann** [comment]
@vlvassilev 

* Added `description` and first `revision`
* Removed info about `ietf-yang-types@2013-07-15.yang` from commit message
* Validated module with `pyang --lint`

**Pull request merged**